### PR TITLE
Updating PVC scenario to add PVC check

### PR DESCRIPTION
**Pull request by shahsahil264**
@chaitanyaenr @paigerube14

### CODE REVIEW

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

File: ci-operator/step-registry/redhat-chaos/pvc-scenarios/prometheus-pod/redhat-chaos-pvc-scenarios-prometheus-pod-commands.sh [hunk 1]
```diff
@@ -54,6 +55,18 @@ export ENABLE_ALERTS=False
...
 export TELEMETRY_PASSWORD=$telemetry_password
 
+#Check if PVC is created
+PVC_CHECK=$(oc get pvc $PVC_NAME -n $TARGET_NAMESPACE --no-headers --ignore-not-found)
+
+if [ -z "$PVC_CHECK" ]; then
+  echo "PVC '$PVC_NAME' does not exist in namespace '$NAMESPACE'."
+  echo "Creating PV and PVC"
+  cluster_monitoring_config
+  echo "Sleeping for 60 seconds for the PV and PVC to be bound"
```
Comment (chaitanyaenr) [inline ci-operator/step-registry/redhat-chaos/pvc-scenarios/prometheus-pod/redhat-chaos-pvc-scenarios-prometheus-pod-commands.sh:65]: We might want to replace sleep with a check that polls every second to see if pv/pvc exists within a certain timeout - 2 mins to make it more robust. Thoughts?

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by paigerube14** [comment]
Looks good, @chaitanyaenr any further thoughts?

**Comment by chaitanyaenr** [comment]
> Looks good, @chaitanyaenr any further thoughts?

LGTM @paigerube14! Nice addition @shahsahil264!

**Review by chaitanyaenr — APPROVED** [review]
/lgtm

**Pull request merged**